haskell/src/Ligo/Util.hs

```diff
@@ -20,7 +20,7 @@ import System.Environment (lookupEnv)
 import System.FilePath ((</>))
 
 import Michelson.Parser
-import Michelson.Test.Import -- TODO 258: remove cleveland dependency
+import Michelson.Runtime.Import (readContract)
```

Comment:
I see that you now import `readContract` here, but one more change that we can do is to use `readValue` as well, which was added in the same [morley MR](https://gitlab.com/morley-framework/morley/-/merge_requests/820) and allows us to not need to implement `readTypedValue_` in this module.

Can you please replace that as well?

Reply:
I've updated the commit message, as well as using the `readValue` instead now.